test/jdk/java/nio/file/Files/probeContentType/Basic.java

```diff
@@ -189,18 +189,8 @@ public static void main(String[] args) throws IOException {
         exTypes.add(new ExType("xlsx", List.of("application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")));
         exTypes.add(new ExType("wasm", List.of("application/wasm")));
 
-        // extensions with content type that differs on Windows 11+ and
-        // Windows Server 2025
-        if (OperatingSystem.isWindows() &&
-            (StaticProperty.osName().matches("^.*[11|2025]$") ||
-                new OSVersion(10, 0).compareTo(OSVersion.current()) > 0)) {
```

Comment:
I assume the import of OSVersion isn't needed now.

Reply:
Removed in 0e0b66a.